package.json

```diff
@@ -16,12 +16,12 @@
     "test-next": "jest ./src/tests/unit/api-next/ --maxWorkers=8 --reporters=default --useStderr --detectOpenHandles",
     "clean": "rm -rf dist",
     "build": "yarn clean && yarn tsc",
+    "build:watch": "yarn tsc --watch",
     "prepublishOnly": "yarn build",
     "release": "yarn lint && yarn test && yarn doc && yarn build",
     "doc": "typedoc --excludeInternal",
     "doc:dev": "typedoc --out ./doc-dev",
-    "tsc": "./node_modules/.bin/tsc",
-    "debug": "ts-node-dev --inspect=4321 --respawn src/tools/account-summary.ts -port=4002 -watch -log=debug"
```

Comment:
debug command is used by docker sample app. 

Please replace the command in `api.Dockerfile` @mfrener

Reply:
Change to run the positions.js on docker https://github.com/stoqey/ib/pull/72/commits/89694134cd5567bdf9e8a5b3ff1d5c2312ba6fb8